src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -487,6 +521,7 @@ protected virtual void CompleteNode(ExecutionContext context, ExecutionNode node
                     }
                     else if (node is ValueExecutionNode valueNode)
                     {
+                        //TODO: need test here as for https://github.com/graphql-dotnet/graphql-dotnet/issues/2348 , exception not always should prevents execution
                         node.Result = valueNode.GraphType.Serialize(node.Result)
```

Comment:
One more `Serialize` call with `?? throw`.

Reply:
> One more `Serialize` call with `?? throw`.

Which is correct at this time.